src/pages/documentation/resources/upgrade-platform-sdks/tabs/analytics.md

```diff
@@ -0,0 +1,245 @@
+
+<Variant platform="ios" task="config" repeat="3"/>
+
+If using Cocoapods, remove the v4 Mobile SDK dependency from the Podfile:
+
+```bash
+target 'YourTarget' do
+    pod 'AdobeMobileSDK'
+    ...
+end
+```
+
+Alternatively, if the v4 Mobile SDK library is linked in Xcode, select the application target and go to `Build Phases`, then `Link Binary With Libraries` and remove `AdobeMobileLibrary.a`.
+
+<Variant platform="android" task="aep-install" repeat="2"/>
+
+#### Java
+
+```java
+import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.Analytics;
+import com.adobe.marketing.mobile.Identity;
+
+@Override
+public void onCreate(Bundle savedInstanceState) {
+  super.onCreate(savedInstanceState);
+  setContentView(R.layout.main);
+
+  MobileCore.setApplication(getApplication());
+
+  MobileCore.registerExtensions(Arrays.asList(
+					Analytics.EXTENSION,
+					Identity.EXTENSION
+				), value -> {
+			// add your Environment file ID from Environments tab in Data Collection tags.
+			MobileCore.configureWithAppID("your-environment-file-id");
+	});
+}
+```
+
+<Variant platform="ios" task="aep-install" repeat="4"/>
+
+#### Swift
+
+```swift
+import AEPCore
+import AEPIdentity
+import AEPAnalytics
+
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+  MobileCore.registerExtensions([Analytics.self, Identity.self], {
+      // Use the environment file id assigned to this application in Data Collection UI
+      MobileCore.configureWith(appId: "your-environment-file-id")
+  })
+  return true
+}
+```
+
+#### Objective-C
+
+```objectivec
+// AppDelegate.h
+@import AEPCore;
+@import AEPIdentity;
+@import AEPAnalytics;
+
+// AppDelegate.m
+- (BOOL) application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+  [AEPMobileCore registerExtensions:@[AEPMobileAnalytics.class, AEPMobileIdentity.class] completion:^{
+      // Use the environment file id assigned to this application in Data Collection UI
+      [AEPMobileCore configureWithAppId: @"your-environment-file-id"];
+  }];
+  return YES;
+}
+```
+
+<Variant platform="android" task="api-changes-v4" repeat="4"/>
+
+#### Java
+
+The Adobe Mobile Library (v4) syntax and usage examples for these API are:
+
+```java
+// syntax
+public static void trackState(final String state, final Map<String, Object> contextData)
+
+// usage
+Analytics.trackState("MainPage", new HashMap<String, Object>() {{
+  put("firstVisit", true);
+}});
+```
+
+```java
+// syntax
+public static void trackAction(final String action, final Map<String, Object> contextData)
+
+// usage
+Analytics.trackAction("linkClicked", new HashMap<String, Object>() {{
+  put("url", "https://www.adobe.com");
+}});
+```
+
+<Variant platform="ios" task="api-changes-v4" repeat="4"/>
```

Comment:
I'm wondering why we need v4 syntax/ examples. We should only provide AEP syntax/ examples for the corresponding APIs, along with specifying any changes to parameter types (like you've already done).

Reply:
this is not new content, just moved from analytics-comparison.md + upgrade to AEP samples